src/MixpanelEventForwarder.js

```diff
@@ -90,9 +90,9 @@ var constructor = function () {
                 reportEvent = true;
                 logEvent(event);
             } else if (event.EventDataType == MessageType.PageView) {
-                event.EventName = 'Viewed ' + event.EventName;
                 reportEvent = true;
                 logEvent(event);
+                logPageView(event);
```

Comment:
`logEvent` needs to be removed, otherwise Mixpanel will get 2 events.  One from `logEvent` and one from `logPageView`.

Reply:
Good catch!